Core/Src/porting/msx/main_msx.c

```diff
@@ -1788,7 +1787,17 @@ void app_main_msx(uint8_t load_state, uint8_t start_paused, uint8_t save_slot)
         drawFrame = common_emu_frame_loop();
         odroid_gamepad_state_t joystick;
         odroid_input_read_gamepad(&joystick);
-        common_emu_input_loop(&joystick, options);
+        void _blit()
```

Comment:
Could you avoid using a nested functions as it's not standard C ?

Reply:
It can be avoided but the alternative is transporting state into statics or refactoring the existing code to expose the state. I suggest we keep the inner function as the code is a lot cleaner. Also we compile for a known target with a known compiler that support inner functions.